docs/scanners/uswds.md

```diff
@@ -1,71 +1,52 @@
-# U.S. Web Design System (USWDS) Scanner
+# U.S. Web Design System Scanner
 
 ## Summary
-A [scan](https://site-scanning.app.cloud.gov/searchUSWDS/) to check each domain for the presence or absence of a USWDS component (i.e. font, style sheet) and the version in use of USWDS.
+The [USWDS scanner](https://site-scanning.app.cloud.gov/searchUSWDS/) checks each domain for the use of [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/) code and the code version.
 
-Specifically, the scan [searches for](https://github.com/18F/domain-scan/blob/tspencer/200scanner/scanners/uswds2.py#L36-L123): presence of `-usa` class in the style sheets, `uswds` in HTML, `.usa-` in HTML, `favicon-57.png` (the USWDS flag) on a website, the fonts used by USWDS (current and former versions) - `Public Sans`, `Source Sans`, and `Merriweather` — `uswds` or `uswds version` or `flavicon-57.png` in CSS. It also searches for the presence of HTML tables, although it uses those as a negative indicator that a site is using USWDS.
+Specifically, the scan [searches for](https://github.com/18F/domain-scan/blob/tspencer/200scanner/scanners/uswds2.py#L36-L123): 
 
-* USWDS adoption
-* USWDS configuration
+* Presence of `-usa` class in CSS
```

Comment:
For consistency with the other points, this might be better as 
> `usa-` in CSS

Note that the string should be `usa-` and not `-usa`.

Reply:
Oh, nm — as I look at the script, it looks like this is actually looking for `usa-` classes _in the HTML markup_ not in the CSS (searching for the `class.*"usa-` regex). This should be
> `usa-`-prefixed classes in HTML

@timothy-spencer can you confirm this?

link: link: https://github.com/18F/domain-scan/blob/8223c7110d5f2bfbb8205379e09b4cea11519983/scanners/uswds2.py#L36

Reply:
Yes, this is something I'm searching for in the main page, not any included CSS.  I'd be happy to plug that into the CSS section too if you like.  Whatever is most useful/effective!